ci-operator/config/openshift/metallb/openshift-metallb-main.yaml

```diff
@@ -42,6 +46,26 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: metallb-l2-e2e-metal-ipi
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NETWORK_TYPE=OpenShiftSDN
+        ENABLE_METALLB_MODE=l2
+        METALLB_REPO=https://github.com/openshift/metallb.git
+    workflow: baremetalds-metallb-e2e
+- as: metallb-bgp-e2e-metal-ipi
```

Comment:
We should probably drop the BGP test from OpenShift CI.  We're not supporting BGP yet.  When we do, it'll be a very different implementation from what we'd be testing right now.

Reply:
Changed.